ocfweb/docs/docs/staff/procedures/restarting-services.md

```diff
@@ -19,3 +21,27 @@ Planned restarts of hypervisors should also be announced on this blog, since
 restarting hypervisors can often take several minutes or more.
 
 [status]: http://status.ocf.berkeley.edu
+
+## Rebooting hypervisors
+
+Rebooting hypervisors is a slightly risky business. Hypervisors aren't
+guaranteed to always reboot without problems. Therefore, you shouldn't reboot
+them unless you can physically access the lab in case problems arise.
+Additionally, this risk is the reason (D)SM permission is normally required to
+reboot hypervisors.
+
+So you've gotten the necessary permission and made a post on the status blog
+(if it's a scheduled restart). What now?
+
+First, gracefully shut down all of the VMs. For planned shutdowns of login
+servers (i.e. tsunami, werewolves and corruption), run the `shutdown` command
+well in advance of the shutdown, so users have impending warning. For other VMs,
+you can shut them each down via `sudo virsh shutdown`.
+
+Be careful to **always shut down firestorm last**. This is because once
+firestorm is shut down, LDAP logins go offline, and the hypervisors can
+thereafter only be logged into via the root account. It's strongly recommended
+to `sudo -i` before shutting down firestorm.
```

Comment:
Thanks for mentioning this, it's definitely a good point to make

Reply:
Even if logins (passwd) are cached, you won't be able to sudo if kerberos is down (auth won't work).